src/css/global/icons.scss

```diff
@@ -283,3 +283,24 @@
 .icon-Binoculars:before {
   content: "\e957";
 }
+.icon-Buy:before {
+  content: "\e958";
+}
+.icon-Cooperative:before {
+  content: "\e959";
+}
+.icon-Filter:before {
+  content: "\e95a";
+}
+.icon-Mentorship:before {
+  content: "\e95b";
+}
+.icon-Message:before {
+  content: "\e95c";
+}
+.icon-Volunteer:before {
```

Comment:
what the heck is up with the new files that end with Zone.identifier? like public/assets/fonts/icons/hylo-evo-icons.eot:Zone.Identifier

Reply:
yeah I don't know what they are for but they came along with the other files I was expecting... ahh it looks like its a browser metadata file added when I downloaded them. I'll delete